### ISSUE TITLE
Different  performance metrics values for same dataframe on Jupyter Notebook

### ISSUE THREAD

**Issue by mvpalheta** [pycaret/pycaret#417]
I ran compare_models twice for the same dataframe with the following settings. Before and after turning off and turning on the jupyter notebook and I got different values ​​for the performance metrics. What happens ??

settings
![image](https://user-images.githubusercontent.com/67707385/89960262-ee148400-dc14-11ea-91be-571c1052a938.png)

Run 1 - Before shutdown
![image](https://user-images.githubusercontent.com/67707385/89960054-46975180-dc14-11ea-88fd-e4b5876dd7fb.png)

Run 2 - After turning off and turning on
![image](https://user-images.githubusercontent.com/67707385/89960104-66c71080-dc14-11ea-951f-22f0c9a82c90.png)

**Comment by pycaret** [comment]
@mvpalheta Can you please provide picture of the setup grid (one that is printed after setup is successfully executed) from both the instances?

**Comment by mvpalheta** [comment]
This is the grid from both instances
![image](https://user-images.githubusercontent.com/67707385/90016801-f30f1d00-dc80-11ea-84ba-937a5a6cabf5.png)

**Comment by Yard1** [comment]
I've noticed it too, on Kaggle. Even though session id is constant, the results are different.

**Comment by mvpalheta** [comment]
I tested it on Google Colab and another jupyter notebook on a remote server from my job

**Comment by pycaret** [comment]
@mvpalheta I don't know if it could be due to remote run. I haven't experienced this in a local environment yet. Can you test in your local environment (ideally with a separate conda environment), If it works there as expected then we will start investigating why it behaves differently with remote run.

**Comment by pycaret** [comment]
@Yard1 This one is about setting seed in a few places where it was missed. I think you have already figured this out?

**Comment by Yard1** [comment]
Yes, I think I got them all.

**Comment by pycaret** [comment]
@mvpalheta 

This issue has been fixed in the `pycaret-nightly` release. Please install pycaret-nightly using `pip install pycaret-nightly`. Feel free to open a new issue, in case the problem persists. This fix will be published to PyPi as `pycaret==2.2.0` on October 28, 2020.

**Comment by mvpalheta** [comment]
Sorry, I forgot to answer. In fact, the problem was with the order of variables in the input dataframe. Every time I tried to run Pycaret, I inserted the variables
in different ordering on the dataframe, due to an earlier variable selection method I use. I noticed that performance only changed for algorithms based on
in trees, therefore, in Python, for these algorithms the order of variables matters. I solved performing the alphabetical ordering of variables before running Pycaret
